docs/data/osdf/install-origin-container.md

```diff
@@ -0,0 +1,353 @@
+user@host $ docker run --rm \
+             --publish 8443:8443 \
+             --publish 8444:8444 \
+             --volume /etc/osdf-origin/issuer.jwk:/etc/pelican/issuer.jwk \
+             --volume /etc/osdf-origin/hostcert.pem:/etc/pelican/certificates/tls.crt \
+             --volume /etc/osdf-origin/hostkey.pem:/etc/pelican/certificates/tls.key \
+             --volume /etc/osdf-origin/config.yaml:/etc/pelican/config.d/99-local.yaml \
+             --volume /mnt/origin:/pelican \
+             --name osdf-origin \
+             hub.opensciencegrid.org/pelican_platform/osdf-origin:latest
+```
+
+
+Validating the Origin
+---------------------
+
+Download a test file (POSIX) or object (S3) from your origin (replacing `ORIGIN_HOSTNAME` with the host name of your origin,
+and TEST_PATH with the OSDF path to the test file or object)
+
+```
+user@host$ curl -L https://ORIGIN_HOSTNAME:8443/TEST_PATH -o /tmp/testfile
+```
+
+Verify the contents of `/tmp/testfile` match the test file or object your origin was serving.
+
+Debugging information will be in your container logs e.g., `docker logs osdf-origin`.
+To increase the debugging information in the origin, edit your origin configuration file and set:
+```
+Debug: true
+```
+
+See [this page](../../common/help.md) for requesting assistance; please include the logs in your request.
+
+
+Joining the Origin to the Federation
+------------------------------------
+
+The origin must be registered with the OSG prior to joining the data federation.
+Send mail to <help@osg-htc.org> requesting registration; provide the following information:
+
+*   Origin hostname
+*   Administrative and security contact(s)
+*   Institution that the origin belongs to
+
+OSG Staff will register the origin and respond with the Resource Name.
+
+Once you have that information, edit `/etc/osdf-origin/config.yaml`, and set `XRootD.Sitename`:
+```
+XRootD:
+  Sitename: <RESOURCE NAME REGISTERED WITH OSG>
+```
+
+Validating the Origin Through the Federation
+--------------------------------------------
+
+Once your origin has been registered in the federation,
+perform the following actions to verify that it can be used through the federation.
+These steps will use the Pelican client
+(which you can download from <https://github.com/PelicanPlatform/pelican/releases>).
+
+If the actions fail, see debugging information in your container logs,
+which are accessible via
+```
+docker logs osdf-origin
+```
+To increase the debugging information in the origin, edit your origin configuration file and set:
+```
+Debug: true
+```
+See [this page](../../common/help.md) for requesting assistance; please include the logs in your request.
+
+1.  Optional, if DirectReads are enabled:  Download a test file (POSIX) or object (S3) directly from your origin,
+    (replacing `<TEST_PATH>` with the OSDF path to the test file or object):
+
+        :::console
+        user@host $ pelican object get 'osdf:///<TEST_PATH>?directread=1' -o /tmp/testfile
```

Comment:
How about we just tell them about `-c https://<origin FQDN>:8443`? That way we don't have to conditionalize this step

Reply:
1. According to the help text, `-c` means cache, so using it on an origin is kind of an abuse of the parameter. Although I do it too, so maybe it's the help text that should be changed, except...

2. If direct reads are off, `-c` shouldn't work either; the fact that it does is kind of bug (https://github.com/PelicanPlatform/pelican/issues/1828).

Reply:
Done